k2/python/csrc/torch/fsa.cu

```diff
@@ -0,0 +1,89 @@
+/**
+ * @brief python wrappers for FSA.
+ *
+ * @copyright
+ * Copyright (c)  2020  Mobvoi Inc.        (authors: Fangjun Kuang)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#include <string>
+#include <utility>
+
+#include "k2/csrc/array.h"
+#include "k2/csrc/fsa.h"
+#include "k2/csrc/fsa_utils.h"
+#include "k2/python/csrc/torch/fsa.h"
+#include "k2/python/csrc/torch/torch_util.h"
+#include "torch/extension.h"
+
+namespace k2 {
+
+static void PybindFsaImpl(py::module &m) {
+  using PyClass = Fsa;
+  py::class_<PyClass> pyclass(m, "Fsa");
```

Comment:
OK, what I was thinking was that we could export the Ragged template (maybe instantiated for a certain set of types) pretty directly, i.e one-for-one its members, and that the Fsa object that appears in Python would actually be a Python object, whose "arcs" member would be the Ragged<Arc> object.  We could perhaps make Fsa one object at the Python level that includes vectors of FSAs, I'm not convinced that it makes sense to have separate types for different levels of vector, we could  even support vector-of-vector-of-FSA which would be useful for n-best lists.

Reply:
Not sure if I understand correctly, supposing there is a peoperty `ndim` in `Fsa` object, then an `Fsa` object with `ndim = n` is a vector of `Fsa`s with `ndim = n-1`, right?

Reply:
I'll make FSA a python class and wrap Ragged<T> instead.

Reply:
Fixed.